ui/src/app/subnets/views/SubnetDetails/StaticRoutes/StaticRoutes.tsx

```diff
@@ -148,15 +151,31 @@ const StaticRoutes = ({ subnetId }: Props): JSX.Element | null => {
   );
   const subnets = useSelector(subnetSelectors.all);
   const subnetsLoading = useSelector(subnetSelectors.loading);
+  const isSuperUser = useSelector(userSelectors.isSuperUser);
   const loading = staticRoutesLoading || subnetsLoading;
+  const [isAddStaticRouteOpen, setIsAddStaticRouteOpen] = useState(false);
```

Comment:
We tend to have one expanded state for add/edit/delete so I'll update that here for when the other forms are added e.g. https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/subnets/components/ReservedRanges/ReservedRanges.tsx#L180